packages/merge/src/components/MergeMatrix/table-components/HeaderRow.js

```diff
@@ -2,7 +2,7 @@ import PropTypes from 'prop-types'
 import React from 'react'
 import {intlShape} from 'react-intl'
 import {Icon} from 'tocco-ui'
-import {stylingPosition} from 'tocco-ui/src/utilStyles'
+import {design} from 'tocco-ui/src/utilStyles'
```

Comment:
if possible dont import with "src" in path

Reply:
see b55a4d7b3305e59436f22b0f8d1543bc0f32adbe